{"name": "test_CodeError", "status": "passed", "description": "验证码输入错误", "attachments": [{"name": "log", "source": "440cfb8a-ac59-422d-be7d-a0aa118d9ba9-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "a1bbe35a-91f2-47b9-9ea0-cefa36d7aa68-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "b2296bdd-e46f-4921-aa49-7f254f73bf00-attachment.txt", "type": "text/plain"}], "start": 1629186620973, "stop": 1629186621158, "uuid": "c616847d-a378-4a83-83c0-b502fd9b5424", "historyId": "c47f0c1c6ff9bd2405cb2124baf4470c", "testCaseId": "84e6eb32be4ef16b824a4ec24731336a", "fullName": "test_case.Register_case.TestRegister#test_CodeError", "labels": [{"name": "story", "value": "参数填写不正确，注册失败"}, {"name": "feature", "value": "注册模块"}, {"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "Register_case"}, {"name": "subSuite", "value": "TestRegister"}, {"name": "host", "value": "DESKTOP-KSOAST3"}, {"name": "thread", "value": "13444-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.Register_case"}]}